.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
```

Comment:
done . no errors

Reply:
I think you forgot to `git push` the changes here?

Reply:
pushed it in another branch named enhancement  should I make a new PR? but not all are finished . I'm doing one commit per changes

Reply:
Ahh... all the requested changes in this branch should be done on this branch then pushed it here. Otherwise, the code reviewer wouldn't be able to see the changes you've made and reviewed it again.

Reply:
can I pushed the changes don on a seperate branch to this branch or should I check out to this branch and redo all the changes?

Reply:
You can do:

// Go to `feature-todo`
git checkout feature-todo

// Get changes from the other branch named `enhancements`?
git merge enhancements

// Push changes to GitHub
git push

Reply:
Awesome! done merging